pkgs/development/python-modules/warp-lang/default.nix

```diff
@@ -351,12 +342,6 @@ buildPythonPackage {
                 writableTmpDirAsHomeHook
               ];
               requiredSystemFeatures = lib.optionals cudaSupport [ "cuda" ];
-              # Many unit tests fail with segfaults on aarch64-linux, especially in the sim
-              # and grad modules. However, other functionality generally works, so we don't
-              # mark the package as broken.
-              #
-              # See: https://www.github.com/NVIDIA/warp/issues/{356,372,552}
-              meta.broken = effectiveStdenv.hostPlatform.isAarch64 && effectiveStdenv.hostPlatform.isLinux;
```

Comment:
Is this no longer the case in `1.9`? Or have the been removed?

Reply:
These issues (NVIDIA/warp#356) have been closed and are included in [milestone 1.9.0](https://github.com/NVIDIA/warp/milestone/7). I haven’t tested them on `aarch64-linux` since I’m only emulating this architecture and cannot reproduce the problems, but all tests pass on `aarch64-darwin`.

From the changelog:

> - Arguments for CPU kernels are now passed as a pointer to a structure to avoid a bug in the ARM64 calling convention implementation of the libffi library ([GH-356](https://github.com/NVIDIA/warp/issues/356)).